src/Razor/src/Microsoft.CodeAnalysis.Razor.Workspaces/ProjectSystem/IProjectSnapshotExtensions.cs

```diff
@@ -33,4 +37,12 @@ public static RazorProjectInfo ToRazorProjectInfo(this IProjectSnapshot project,
             projectWorkspaceState: project.ProjectWorkspaceState,
             documents: documents.DrainToImmutable());
     }
+
+    public static ImmutableArray<TagHelperDescriptor> GetTagHelpersSynchronously(this IProjectSnapshot projectSnapshot)
```

Comment:
Would you mind creating an issue to clean this up? I'm about to work on `VisualStudioDocumentTracker` as part of the MEF clean up, and I'd be happy to get rid of this at the same time.